pkgs/development/embedded/arduino/arduinoOTA/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+buildGoModule rec {
+  pname = "arduinoOTA";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "arduino";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-HaNMkeV/PDEotYp8+rUKFaBxGbZO8qA99Yp2sa6glz8=";
+  };
+
+  vendorHash = null;
+
+  postPatch = ''
+    sed -i 's/\(versionString        = "\)/\1${version}/' version/version.go
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    homepage = "https://github.com/arduino/arduinoOTA";
+    description = "arduinoOTA is a tool for uploading programs to Arduino boards over a network.";
```

```suggestion
    description = "A tool for uploading programs to Arduino boards over a network";
```
missed this the first time, in nixpkgs descriptions the name of the program shouldn't be repeated and sentences are supposed to end without punctuation.